src/test/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRowHandlerTest.java

```diff
@@ -66,7 +66,7 @@ DiscreteGroundWater makeDgw() {
 		dgw.measurementSourceCode = "";
 		dgw.measuringAgencyCode = "USGS";
 		dgw.levelAccuracyCode = "2"; // two digits after decimal point
-		dgw.siteStatusCode = ""; // R, S or blank
+		dgw.readingQualifiers = ""; //Numeric code
```

Comment:
Wait, isn't this a JSON object, not a numeric?

Reply:
Oh, now I get it -- because of unit tests.

Reply:
@duselmann I was confused initially too, but I think NWISWeb expects a numeric code here, so it's just mapped based on those incoming quals.